### DISCUSSION TITLE
CHANGE HOME ICON

### DISCUSSION THREAD

**Discussion by cristiandlahoz** [saicaca/fuwari#225] [Q&A]
![image](https://github.com/user-attachments/assets/ac541f80-a47b-444a-b41b-3d354d3aaaf9)

Hey, how can I change the home icon in the NavBar?

**Comment by saicaca** [comment]
There's no configuration for this. So you need to edit it here.

https://github.com/saicaca/fuwari/blob/ff9e82091a47b619ce4abaed410f7dd2a25db58d/src/components/Navbar.astro#L29

**Reply by cristiandlahoz** [reply]
thanks!
